docs/es-overview.asciidoc

```diff
@@ -2,169 +2,38 @@
-** https://www.elastic.co/integrations?solution=security[Beat modules]: {beats}
-are lightweight data shippers. Beat modules provide a way of collecting and
-parsing specific data sets from common sources, such as cloud and OS events,
-logs, and metrics. Common security-related modules are listed
-<<enable-beat-modules, here>>.
-* The {security-app} in {kib} is used to manage the *Detection engine*,
-*Cases*, and *Timeline*, as well as administer hosts running {elastic-defend}:
-** Detection engine: Automatically searches for suspicious host and network
-activity via the following:
-*** <<detection-engine-overview, Detection rules>>: Periodically search the data
-({es} indices) sent from your hosts for suspicious events. When a suspicious
-event is discovered, an alert is generated. External systems, such as
-Slack and email, can be used to send notifications when alerts are generated.
-You can create your own rules and make use of our <<prebuilt-rules, prebuilt ones>>.
-*** <<detections-ui-exceptions, Exceptions>>: Reduce noise and the number of
-false positives. Exceptions are associated with rules and prevent alerts when
-an exception's conditions are met. *Value lists* contain source event
-values that can be used as part of an exception's conditions. When
-{elastic-defend} is installed on your hosts, you can add malware exceptions
-directly to the endpoint from the Security app.
-*** <<included-jobs, {ml-cap} jobs>>: Automatic anomaly detection of host and network events. Anomaly scores are provided per host and can be used with detection rules.
-** <<timelines-ui, Timeline>>: Workspace for investigating alerts and events.
-Timelines use queries and filters to drill down into events related to
-a specific incident. Timeline templates are attached to rules and use predefined
-queries when alerts are investigated. Timelines can be saved and shared with
-others, as well as attached to Cases.
-** <<cases-overview, Cases>>: An internal system for opening, tracking, and sharing
-security issues directly in the Security app. Cases can be integrated with
-external ticketing systems.
-** <<admin-page-ov, Administration>>: View and manage hosts running {elastic-defend}.
+[[siem-integration]]
+=== Learn more
 
-<<ingest-data>> and <<install-endpoint>> describe how to ship security-related
-data to {es}.
+To learn more about {elastic-sec}, use the left-side navigation menu, review the <<getting-started, Getting started>> documentation section (which covers topics including system requirements, workspaces, how to configure {elastic-sec}, and how to ingest data), or refer to the following resources:
 
+* <<es-ui-overview, Elastic Security UI overview>> — learn about navigating among {elastic-sec}'s various tools and interfaces.
```

Comment:
Not sure why this is displaying as an em dash in the HTML output, or if this fix will take. 

```suggestion
* <<es-ui-overview, Elastic Security UI overview>> - learn about navigating among {elastic-sec}'s various tools and interfaces.
```